src/stylesheets/button.scss

```diff
@@ -1,12 +1,17 @@
 button.btn, a.btn {
   display: block;
   color: $white;
-  &:not(.btn-danger) {
-    background: $blue;
-  }
+  transition: background .5s, color .5s;
+  background: $blue;
+  margin: .5em;
+  border-radius: .25em;
+  font-size: 16px;
   &.btn-danger {
```

Comment:
I know this was here before, but why isn't this `danger` just like how `success` is `success` and not `btn-success`?

Reply:
Because we were originally pulling the styling from Bootstrap, where the class name is `btn-danger`.  (But yes, that should be updated now; good catch!)